tools/packaging/static-build/shim-v2/build.sh

```diff
@@ -22,15 +24,12 @@ DESTDIR=${DESTDIR:-${PWD}}
 PREFIX=${PREFIX:-/opt/kata}
 container_image="${SHIM_V2_CONTAINER_BUILDER:-$(get_shim_v2_image_name)}"
 
-EXTRA_OPTS="${EXTRA_OPTS:-""}"
-VMM_CONFIGS="qemu fc"
-REMOVE_VMM_CONFIGS="${REMOVE_VMM_CONFIGS:-""}"
-
```

Comment:
@GeorginaKin - we need all these lines still, so if you can add them back in, that would be great.